sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -170,6 +170,24 @@ static const struct snd_soc_acpi_adr_device rt1316_1_single_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt712_0_single_adr[] = {
+	{
+		.adr = 0x000030025D071201ull,
+		.num_endpoints = 1,
+		.endpoints = &single_endpoint,
```

Comment:
we're not using the endpoints for now, but I don't think this is correct. We have two with headphone/jack and amplifier, no?

Reply:
> we're not using the endpoints for now, but I don't think this is correct. We have two with headphone/jack and amplifier, no?

Added rt712_endpoints[] for rt712.